Also publish flow typings to npm

wietsevenema opened the pull request:
Also publish the flow typings to npm, so we can use them. 

Where should I add the documentation on how to use this? You should add this to the `[libs]` section of your .flowconfig. 
```
[libs]
node_modules/mobx/lib/mobx.js.flow
```

mweststrate commented:
@wietsevenema could you add that to the readme, probably somewhere near the installation notes. Thanks!

wietsevenema commented:
- Added documentation next to Bower support.
 - Rewrote using new dependency: ncp, to keep the build windows friendly. Had to after #770 was fixed.

mweststrate commented:
Awesome, thanks a lot!
